### DISCUSSION TITLE
ISO_LEVEL5_Shift also works

### DISCUSSION THREAD

**Discussion by morenathan** [YaLTeR/niri#890] [Ideas]
I know we've discussed this possibility in the past, but it came down to modifying your input library. The Wiki doesn't mention ISO_LEVEL5_Shift, but now it works. Maybe the library changed and this was missed in the documentation?

Either way it makes me estactic since I have it mapped to right-control. I didn't see a way to change the page.

**Comment by YaLTeR** [comment]
It was added in https://github.com/YaLTeR/niri/pull/565. I guess I forgot to change the wiki. Do you mind making a pull request to the file in the wiki/ folder?

**Reply by morenathan** [reply]
Okay, request sent. Documentation is a must. So if I catch anything else will just submit it. Haven't done that before.
